bigchaindb/commands/bigchaindb.py

```diff
@@ -242,4 +242,7 @@ def create_parser():
 
 
 def main():
-    utils.start(create_parser(), sys.argv[1:], globals())
+    if sys.version_info[0] == 3 and sys.version_info[1] <= 5:
+        logger.info('Please use python version 3.6 or up')
```

Comment:
1. I'd write '...version 3.6 or higher'
1. What's the default log level? (I wonder if a default user will see this message.)
1. Could we just exit (shut down BigchainDB Server) if they're trying to use Python 3.5 or lower? If they are, it won't work, and they'll get a big long confusing message (in addition to the new and actually-helpful message).